standard/tier/tier_data.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			order = 1,
+			value = '1',
+			sort = 'A1',
+			name = 'S-Tier',
+			short = 'S',
+			link = 'S-Tier Tournaments',
+			category = 'S-Tier Tournaments',
+		},
```

Comment:
```suggestion
		{
			tier = 1,
			name = 'S-Tier',
			short = 'S',
			link = 'S-Tier Tournaments',
		},
```

with `data.index`, `data.storageValue`, and `data.category` auto-computed in `Module:Tier`

Reply:
- categories "can" differ from the links, so you can not auto calculate them
- index/order can not be calculated for tiertypes
- value = tier ? (just as string)
- sort differs from wiki to wiki and can also not be easily calculated (especially if there are > 10 tiertypes)

Reply:
order kicked, rest stays due to above reasons

Reply:
> * categories "can" differ from the links, so you can not auto calculate them
> * index/order can not be calculated for tiertypes
> * value = tier ? (just as string)
> * sort differs from wiki to wiki and can also not be easily calculated (especially if there are > 10 tiertypes)
> 
> could kick order though

- yea category defaults to the link, can still be specified.
- I still don't understand why tiertype can't be a separate enum. Maybe I'm not well versed in the use cases across wikis. Are there any cases where tier and tiertype order are interleaved? instead of tiers first, then tiertypes
- I renamed `data.value` to `data.storageValue`, which defaults to `tostring(data.tier)`. `data.tier` is the actual tier